docs/DEPLOYMENT.md

```diff
@@ -165,3 +165,17 @@ Check if it complains about any missing environment variables or if there's any
 If you can't figure out what's happening, please [open an issue here](https://github.com/briefercloud/briefer/issues).
 
 </details>
+
+<details>
+  <summary>I can access the web application, but sign-in is not working</summary>
+
+If you're running Briefer from another machine - for example an external server, Raspbery Pi etc - make sure you have enabled HTTPS for your current setup while accessing Briefer's application on port 3000.
+
+In case you can't enable HTTPS or just want to try it out, you can run Briefer in an insecure manner by setting the `ALLOW_HTTP` environment variable to `true`:
+
+```sh
+user@server:~$ ALLOW_HTTP=true briefer
```

Comment:
```suggestion
ALLOW_HTTP=true briefer
```

We usually don't add the `$` in front of code blocks so it's easier to copy and paste

Reply:
Sounds good - it just looked nicer :P